spec/features/03_cart_spec.rb

```diff
@@ -0,0 +1,67 @@
+require_relative '../spec_helper'
+
+feature 'RSpec: practical task_2, Rozetka UI Testing', type: :feature do
+  include Components
+  describe 'When to Cart added one product' do
+    before(:each) do
+      visit('/')
+      @search = SearchPage.new
+      @product = CartPage.new
+      @search.search_product('Amazfit GTR')
+      click_link(ProductsConstants::WATCH)
+      click_button('Купить')
+    end
+
+    it 'Verifies add product to the cart' do
+      expect(get_value_from_element(Tags::TOTAL_PRICE)).to eq('3474')
+    end
+
+    it 'Verifies update product quantity in the cart' do
+
+      find(CartLocators::PLUS_ONE).click
+      sleep 2
+      expect(get_value_from_element(Tags::TOTAL_PRICE)).to eq('6948')
+    end
+
+    it 'Verifies remove product from the cart' do
+
+      @product.remove_from_cart(ProductsConstants::WATCH)
+      sleep 2
+      pending("something else getting finis")
```

Comment:
this comment was as an example, you can provide here details related to your scenario